src/scripts/services/group-service.ts

```diff
@@ -1,7 +1,8 @@
 import * as okta from '@okta/okta-sdk-nodejs';
-import * as TE from 'fp-ts/lib/TaskEither';
-import * as O from 'fp-ts/lib/Option';
-import { pipe } from 'fp-ts/lib/function';
+import * as TE from 'fp-ts/TaskEither';
+import * as O from 'fp-ts/Option';
+import { constant, pipe } from 'fp-ts/function';
+import { type UserIdOption } from '../list-groups';
```

Comment:
**Issue:** services should not depend on scripts - we want services to be unaware of any particulars related to scripts. Either:
* the type moves into the service;
* the type is replaced with an `Option<UserId>`; or
* the method is split into two `listGroups` and `listUserGroups`.

My slight preference is for the third option.

Reply:
Have gone with the third option and removed the import.